tests/filecheck/mlir-conversion/with-mlir/dialects/emitc/emitc_types.mlir

```diff
@@ -16,3 +20,20 @@
   i32_1D = !emitc.array<10xi32>,
   index_1D = !emitc.array<1xindex>
 }: ()->()
+
+//===----------------------------------------------------------------------===//
+// LValueType
+//===----------------------------------------------------------------------===//
+
+// CHECK: f64 = !emitc.lvalue<f64>
+// CHECK-SAME: i32 = !emitc.lvalue<i32>
+// CHECK-SAME: index = !emitc.lvalue<index>
+// CHECK-SAME: tensor_i32 = !emitc.lvalue<tensor<1xi32>>
+"test.op"() {
+  f64 = !emitc.lvalue<f64>,
+  i32 = !emitc.lvalue<i32>,
+  index = !emitc.lvalue<index>,
+  tensor_i32 = !emitc.lvalue<tensor<1xi32>>
+  // ptr_i32 = !emitc.lvalue<!emitc.ptr<i32>>,
```

Comment:
What's the deal with these commented lines? Can you please add a comment here and in the other test indicating what the plan for them is and why they're commented?

Reply:
I added a comment, would it be better to create an issue and reference it there?

Reply:
In this case I think comment is fine